### PR TITLE
[onert] Add overflow check in MinMaxReader

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit add num_run, num_op, num_input overflow check in MinMaxReader.

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

For clearing code analysis defect warning

### CODE REVIEW

**Comment by hseok-oh** [comment]
To clear code analysis defect warning

File: runtime/onert/odc/MinMaxReader.cc [hunk 1]
```diff
@@ -99,6 +100,10 @@ MinMaxVectors MinMaxReader::readOP(uint32_t model_idx, uint32_t subg_idx, uint32
   float minmax[2];
   const int64_t data_size = sizeof(float) * 2 + sizeof(uint32_t) * 3;
 
+  // Check num_run overflow
+  if (num_run > std::numeric_limits<int32_t>::max() / data_size)
```
Comment (ragmani) [inline runtime/onert/odc/MinMaxReader.cc:104]: Can I get the reason why the max value of `int32_t` is used?

Comment (hseok-oh) [reply runtime/onert/odc/MinMaxReader.cc]: I've updated to `uint32_t`.

Comment (ragmani) [reply runtime/onert/odc/MinMaxReader.cc]: I was curious about how to check for overflow with `std::numeric_limits<int32_t>::max() / data_size`.
Anyway, adding code to check for overflow looks good.

**Review by ragmani — APPROVED** [review]
LGTM

File: runtime/onert/odc/MinMaxReader.cc (hunk 1 above)
Comment (hseok-oh) [reply runtime/onert/odc/MinMaxReader.cc]: Analyzer required to check upper limit for loop condition from file reading. So I added upper limit based on file size limit.

**Pull request merged**